skopt/optimizer/forest_opt.py

```diff
@@ -0,0 +1,167 @@
+"""Forest based minimization algorithms."""
+
+from sklearn.utils import check_random_state
+
+from .base import base_minimize
+from ..learning import ExtraTreesRegressor
+from ..learning import RandomForestRegressor
+
+
+def forest_minimize(func, dimensions, base_estimator=None,
```

Comment:
The old default was `base_estimator='et'` this is what is making travis unhappy. I think I prefer having `'et'` explicitly instead of using `None`. WDYT?